vochain/account.go

```diff
@@ -32,15 +34,18 @@ func (a *Account) Unmarshal(data []byte) error {
 }
 
 // Transfer moves amount from the origin Account to the dest Account.
-func (a *Account) Transfer(dest *Account, amount uint64, nonce uint32) error {
+func (a *Account) Transfer(dest *Account, amount uint64, nonce uint32, faucet bool) error {
```

Comment:
Umm, I don't like this approach (personally). I'd prefer to let Transfer() as a method for transfering balance among accounts and create a new method under `Account` named something like `CollectFaucet()`.

The reason is that Transfer is a very important method that will be used by several parts of the code. Not all developers will understand what the "faucet" boolean means and its easy to make errors (and important ones).